src/coreclr/vm/exkind.h

```diff
@@ -0,0 +1,28 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// RuntimeExceptionKind.h
+//
+
+#ifndef __exkind_h__
+#define __exkind_h__
```

Comment:
`__` convention is for system headers. While we (incorrectly) use this convention in places, @janvorli has pointed out a few times that we should be using `HAVE_<name>` instead.

```suggestion
#ifndef HAVE_EXKIND_H
#define HAVE_EXKIND_H
```